x-pack/plugin/frozen-indices/src/internalClusterTest/java/org/elasticsearch/index/engine/frozen/FrozenIndexTests.java

```diff
@@ -120,48 +120,66 @@ public void testCloseFreezeAndOpen() throws Exception {
         assertTrue(indexService.getIndexSettings().isSearchThrottled());
 
         // now scroll
-        SearchResponse searchResponse = client().prepareSearch()
-            .setIndicesOptions(IndicesOptions.STRICT_EXPAND_OPEN_FORBID_CLOSED)
-            .setScroll(TimeValue.timeValueMinutes(1))
-            .setSize(1)
-            .get();
-        do {
-            assertHitCount(searchResponse, 3);
-            assertEquals(1, searchResponse.getHits().getHits().length);
-            SearchService searchService = getInstanceFromNode(SearchService.class);
-            assertThat(searchService.getActiveContexts(), Matchers.greaterThanOrEqualTo(1));
-            for (int i = 0; i < 2; i++) {
-                shard = indexService.getShard(i);
-                engine = IndexShardTestCase.getEngine(shard);
-                // scrolls keep the reader open
-                assertTrue(((FrozenEngine) engine).isReaderOpen());
-            }
-            searchResponse = client().prepareSearchScroll(searchResponse.getScrollId()).setScroll(TimeValue.timeValueMinutes(1)).get();
-        } while (searchResponse.getHits().getHits().length > 0);
-        client().prepareClearScroll().addScrollId(searchResponse.getScrollId()).get();
-
-        String pitId = openReaders(TimeValue.timeValueMinutes(1), indexName);
+        SearchResponse scrollResponse = null;
         try {
-            for (int from = 0; from < 3; from++) {
-                searchResponse = client().prepareSearch()
-                    .setIndicesOptions(IndicesOptions.STRICT_EXPAND_OPEN_FORBID_CLOSED)
-                    .setPointInTime(new PointInTimeBuilder(pitId))
-                    .setSize(1)
-                    .setFrom(from)
-                    .get();
-                assertHitCount(searchResponse, 3);
-                assertEquals(1, searchResponse.getHits().getHits().length);
-                SearchService searchService = getInstanceFromNode(SearchService.class);
-                assertThat(searchService.getActiveContexts(), Matchers.greaterThanOrEqualTo(1));
-                for (int i = 0; i < 2; i++) {
-                    shard = indexService.getShard(i);
-                    engine = IndexShardTestCase.getEngine(shard);
-                    assertFalse(((FrozenEngine) engine).isReaderOpen());
+            scrollResponse = client().prepareSearch()
```

Comment:
If we actually want to adjust this one here, why don't we start from the standard `assertResponse` approach here. why use a tricky try-finally and deviatee from our usual approach?